### PR TITLE
Teach `lite` to infer the `silt` binary location

### PR DESCRIPTION
**Pull request by harlanhaskins**
### What's in this pull request?

This removes the need to pass the `--silt <path-to-silt>` command-line option by getting the path of the `lite` executable and looking relative to it for `silt`.

### Why merge this pull request?

It removes an extra level of burden when trying to invoke `lite` manually.

### What's worth discussing about this pull request?

It's worth taking a look at the method of discovering the binary path. This is the same way that `SwiftSyntax` looks for the `swiftc` executable.

### What downsides are there to merging this pull request?

None.

### CODE REVIEW

**Comment by harlanhaskins** [comment]
@CodaFi Mergeable?

File: .travis.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ matrix:
       script:
         - swiftlint
         - swift test
-        - .build/debug/lite --test-dir Tests/Lite --silt .build/debug/silt
+        - .build/debug/lite -d Tests/Lite
```
Comment (CodaFi) [inline .travis.yml]: Does `swift run` work here?

Comment (harlanhaskins) [reply .travis.yml]: It does! I forgot about this 🎉

**Comment by harlanhaskins** [comment]
Looks like Rainbow pushed a build error. I’m gonna file an issue.

**Comment by harlanhaskins** [comment]
Ooh, it was your PR, @CodaFi

**Review by CodaFi — APPROVED** [review]
Looks fantastic.

**Comment by harlanhaskins** [comment]
macOS bots are backed up. 🚢

**Pull request merged**